v2/api_error.go

```diff
@@ -0,0 +1,185 @@
+package gax
+
+// code snippet credit to @ndietz
```

Comment:
You can remove this :)

Reply:
I spot checked a couple of these comments and it seems not all of the resolved comments are actually fixed. Did you forget to push perhaps?